UI/Web/src/app/book-reader/_components/reader-settings/reader-settings.component.html

```diff
@@ -70,6 +70,15 @@ <h2 class="accordion-header">
                         <span class="phone-hidden">&nbsp;{{readingDirectionModel === ReadingDirection.LeftToRight ? 'Left to Right' : 'Right to Left'}}</span>
                     </button>
                 </div>
+                <div class="controls" style="display: flex; justify-content: space-between; align-items: center; ">
+                    <label id="writingstyle" class="form-label">Writing Style <i class="fa fa-info-circle" aria-hidden="true" placement="top" [ngbTooltip]="writingStyleTooltip" role="button" tabindex="0" aria-describedby="writingStyle-help"></i></label>
+                    <ng-template #writingStyleTooltip>Changes the direction of the text. Horizontal is left to right, vertical is top to bottom.</ng-template>
```

Comment:
This seems non-intuitive for first time user, but i can't think of a better description right now.